tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -17,3 +17,11 @@ tests:
     harness_config:
       fixture: spi_loopback
     platform_allow: frdm_k64f
+  drivers.stm32_spi_dma.loopback:
+    tags: dma
+    harness: ztest
+    extra_args: OVERLAY_CONFIG="overlay-stm32-spi-dma.conf"
+    harness_config:
+      fixture: spi_loopback
+    platform_allow: nucleo_g474re nucleo_f207zg nucleo_f429zi nucleo_wb55rg
+        nucleo_f746zg nucleo_g474re nucleo_wb55rg
```

Comment:
nucleo_g474re and nucleo_wb55rg are duplicated

Reply:
remove extra board name